test/demo/utils/notificationsRemote.ts

```diff
@@ -58,17 +59,17 @@ export async function toggleNotificationCenter(executionTarget: Identity) {
 }
 
 export async function addEventListener
-    <E extends NotificationEvent>(executionTarget: Identity, eventType: E['type'], listener: (event: E) => void): Promise<void> {
+    <E extends Events>(executionTarget: Identity, eventType: E['type'], listener: (event: E) => void): Promise<void> {
     const remoteFn = await ofBrowser.getOrMountRemoteFunction(executionTarget, listener);
-    await ofBrowser.executeOnWindow(executionTarget, function(eventTypeRemote: typeof eventType, listenerRemote: typeof listener) {
+    await ofBrowser.executeOnWindow(executionTarget, function(eventTypeRemote: any, listenerRemote: any) {
```

Comment:
@pjbroadbent, any idea why these need to be `any` when they didn't before?

Reply:
TypeScript picks up `typeof eventType` as being a union of strings, which for some reason it can't map to the three overrides of `addEventListener`.

Not entirely sure why this only became an error with this PR. Possibly a side-effect of the change in event definitions.

Reply:
I'll be honest, I'm not especially happy with this change, but I appreciate that the api changes need to go in and it may not be worth the effort of figuring this out right now. At a minimum, can we have a spin-off story to figure this out and fix it. Also, if the `listenerRemote` is now any, then we shouldn't need the `as any as typeof listener` a couple lines below right?

Reply:
@tomrobertsOF Will leave this for you to resolve